hazelcast/src/main/java/com/hazelcast/jet/core/Edge.java

```diff
@@ -711,13 +711,16 @@ public int getClassId() {
         FANOUT
     }
 
-    private static class Single implements Partitioner<Object> {
+    public static class Single implements Partitioner<Object>, IdentifiedDataSerializable {
```

Comment:
This class is public API - we need to move these out (ideal) or mark them as `@PrivateApi` (acceptable)